proxmox/cluster/metrics/server_options.go

```diff
@@ -0,0 +1,47 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package metrics
+
+// ServerData contains the data from a metrics server response.
+type ServerData struct {
+	Disable *int64  `json:"disable,omitempty" url:"disable,omitempty"`
+	ID      *string `json:"id,omitempty"      url:"id,omitempty"`
+	MTU     *int64  `json:"mtu"               url:"mtu,omitempty"`
+	Port    *int64  `json:"port"              url:"port"`
+	Server  *string `json:"server"            url:"server"`
+	Timeout *int64  `json:"timeout,omitempty" url:"timeout,omitempty"`
```

Comment:
`ID`, `Port`, and `Server` are required fields in the API, I'd rather make them non-pointers then.

Reply:
you are right, that makes sense, this is now fixed and ValueString etc used instead of ValueStringPointer etc